### PR TITLE
Urc dev

### PR DESCRIPTION
**Pull request by itsUrcute**
made changes to `tsconfig.json` to use `require()` syntax instead of `import` on compilation.
Changed structure of `config` from `.ts` to `.json`.
re-linted everything and cleaned up `commandHandler.ts`.
changed constants structure from `constants/file.ts` to `constants.ts`.

### CODE REVIEW

**Comment by cheeze2000** [comment]
`constants` should be a folder. might put stuff like hardcoded string messages later on. right now, you can refactor into `constants/guildIds.ts` or `constants/guildIds.json`. in the future, `constants/messages.json` might be there

File: src/constants/guilds.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export const BSguildID = "844907047417675797";
```
Comment (cheeze2000) [inline src/constants/guilds.ts:1]: maybe `BS_GUILD_ID`

File: src/index.ts [hunk 2]
```diff
@@ -1,9 +1,16 @@
...
+import { token } from "../config.json";
 
-const client = new Client(options);
+const client = new Client({
+  intents: [
+    Intents.FLAGS.GUILDS,
+    Intents.FLAGS.GUILD_EMOJIS,
+    Intents.FLAGS.GUILD_MEMBERS,
+    Intents.FLAGS.GUILD_MESSAGES,
+    Intents.FLAGS.DIRECT_MESSAGES
```
Comment (cheeze2000) [inline src/index.ts:11]: are we gonna use guild emojis? dms?

File: src/internals/commandHandler.ts [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import { ApplicationCommand, Collection, GuildApplicationCommandPermissionData, Snowflake } from "discord.js";
 import { readdir } from "fs/promises";
-import { guildID } from "../constants/guildID.js";
+import { BSguildID } from "../constants/guilds";
```
Comment (cheeze2000) [inline src/internals/commandHandler.ts:3]: change this according to the previous comment

File: tsconfig.json [hunk 4]
```diff
@@ -2,24 +2,23 @@
   "$schema": "https://json.schemastore.org/tsconfig",
   "display": "Node 14",
   "compilerOptions": {
-    "target": "ES6",
-    "module": "esnext",
+    "target": "ES2020",
+    "module": "commonjs",
     "declaration": true,
-    "rootDir": "./src",
+    "rootDir": ".",
```
Comment (cheeze2000) [inline tsconfig.json:8]: remove this line, use the default `rootDir`

**Review by cheeze2000 — CHANGES_REQUESTED** [review]

File: src/index.ts (hunk 2 above)
Comment (TogarashiPepper) [reply src/index.ts:11]: I'm fairly sure that we dont need any of these

**Review by cheeze2000 — APPROVED** [review]

**Pull request merged**